nix/pkgs/cc-test-reporter/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  inherit (stdenv) isLinux isDarwin isWindows;
+
+  platform =
+    if isLinux then "linux" else
+    if isDarwin then "darwin" else
+    if isWindows then "windows" else
+    throw "Unsupported platform: ${stdenv.hostPlatform.system}";
+
+in stdenv.mkDerivation rec {
+  pname = "cc-test-reporter";
+  version = "0.11.1";
+
+  src = fetchurl {
+    url = "https://codeclimate.com/downloads/test-reporter/test-reporter-${version}-${platform}-amd64";
+    hash = lib.getAttr platform {
+      darwin = "sha256-uO9aRL3cJe+KCoC+uN6cBQy8xGQHim6h5Qzw36QO7EY=";
+      linux = "sha256-ne79mW3w9tHJ+3lAWzluuRp6yjWsy4lpdV/KpmjaTa0=";
+      windows = "sha256-8pn8csW9l5xMerZWAwIwWcrO7OLNWEM03yPEMMllaak=";
+    };
+   };
```

Comment:
might need tweaking since the build url is slightly different
ref -> https://github.com/codeclimate/test-reporter?tab=readme-ov-file#download

![Screenshot 2024-05-06 at 8 17 28 PM](https://github.com/status-im/status-go/assets/64726664/de32492a-6ff3-4090-96e4-e2ea9dc2ee02)

Reply:
linux has both `amd64` and `amr64` options whereas `windows` and `Darwin` does not.
weird.